<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drawing App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_Main">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_TopControls">
      <item>
       <widget class="QLabel" name="labelGridSize">
        <property name="text">
         <string> Grade:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gridSizeComboBox">
        <property name="toolTip">
         <string>Select the grid dimensions for the drawing area</string>
        </property>
        <item>
         <property name="text">
          <string>4x4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5x5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6x6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7x7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8x8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9x9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10x10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11x11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12x12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13x13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14x14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15x15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16x16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128x128</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_Separator1">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelColor">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Preto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Branco</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CinzaClaro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cinza</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marron</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vermelho</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Laranja</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Amarelo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VerdeClaro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Verde</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cyano</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AzulClaro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Azul</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Roxo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magenta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rosa</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="toolTip">
         <string>Save the current drawing state</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset theme="document-save"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="toolTip">
         <string>Load a previously saved drawing state</string>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="areaDesenho">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_DrawingArea">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="drawingAreaWidgetPlaceholder" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Drawing Area</string>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_BottomControls">
      <item>
       <widget class="QPushButton" name="importButton">
        <property name="toolTip">
         <string>Import data/image into the drawing area</string>
        </property>
        <property name="text">
         <string>Import</string>
        </property>
        <property name="icon">
         <iconset theme="document-import"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_Bottom">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
        <property name="toolTip">
         <string>Export the drawing area contents</string>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
        <property name="icon">
         <iconset theme="document-export"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
